readme: add sunset announcement (#923)
fal-ai/dbt-fal: README.md

  </a>
</p>

# 🌅 Sunset Announcement

> As of April 2024, **dbt-fal** has been officially sunsetted. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. While it's always tough to say goodbye, we're proud of what we've accomplished together.

Author: ```suggestion
As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating the [world's first generative media platform specifically designed for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. While it's always tough to say goodbye, we're proud of what we've accomplished together.
```

> As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating [the world's first generative media platform for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. While it's always tough to say goodbye, we're proud of what we've accomplished together.

Author: ```suggestion
As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating [the world's first generative media platform for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. Together, we've achieved remarkable things, and we look forward to continuing this journey of innovation with you.
```

> As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating the [world's first generative media platform specifically designed for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. While it's always tough to say goodbye, we're proud of what we've accomplished together.

Author: ```suggestion
As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating [the world's first generative media platform specifically designed for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. While it's always tough to say goodbye, we're proud of what we've accomplished together.
```

> As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating [the world's first generative media platform specifically designed for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. While it's always tough to say goodbye, we're proud of what we've accomplished together.

Author: ```suggestion
As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating [the world's first generative media platform for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. While it's always tough to say goodbye, we're proud of what we've accomplished together.
```

> As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating [the world's first generative media platform for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. Together, we've achieved remarkable things, and we look forward to continuing this journey of innovation with you.

Reviewer: Hey everyone! Just wanted to drop in and share some news: as of April 2024, we’re saying goodbye to dbt-fal. Yep, it’s been quite the ride, but we’re switching gears to pour all our energy into something super exciting – creating the first-ever generative media platform for developers over at [fal.ai](https://fal.ai/)! 🚀 We’re all in on this and can’t wait to see where it takes us.

Big thanks to every single one of you who’s been with us on the dbt-fal adventure. Your support and contributions mean the world. We’ve done some awesome stuff together, and this isn’t the end. Just a new chapter. So, here’s to more amazing things ahead, and we’re stoked to have you join us for the ride. Cheers!

> As of April 2024, **dbt-fal** has been officially sunsetted. This strategic pivot allows our team to focus on creating [the world's first generative media platform for developers](https://fal.ai) — a venture we believe in passionately. This decision was made after careful consideration, and we want to extend our sincerest gratitude to all our users and contributors who have been part of this journey. Together, we've achieved remarkable things, and we look forward to continuing this journey of innovation with you.

Author: ```suggestion
**Hey everyone!**

Just wanted to drop in and share some news: as of April 2024, we’re saying goodbye to dbt-fal. Yep, it’s been quite the ride, but we’re switching gears to pour all our energy into something super exciting – creating the first-ever generative media platform for developers over at [fal.ai](https://fal.ai)! 🚀 We’re all in on this and can’t wait to see where it takes us.

**Big thanks** to every single one of you who’s been with us on the dbt-fal adventure. Your support and contributions mean the world. We’ve done some awesome stuff together, and this isn’t the end. Just a new chapter. So, here’s to more amazing things ahead, and we’re stoked to have you join us for the ride.

Cheers!
```

## ❗ What Does This Mean?

- **No Further Development:** The project will no longer receive updates or new features.

> - **Security Vulnerabilities:** We will not be addressing new security vulnerabilities after April 8 2024. We advise users to consider this when deciding to continue the use of the project.

Author: ```suggestion
- **Security Vulnerabilities:** We will not be addressing new security vulnerabilities after April 8, 2024. We advise users to consider this when deciding to continue the use of the project.
```

- **Archival:** The repository will be archived, making it read-only. While the code will remain accessible for educational and historical purposes, we encourage users to fork the repository if they wish to continue development on their own.

## 💬 FAQ

### Can I still use dbt-fal?
Yes, the project will remain available for use, but please be aware that no new updates or security patches will be provided moving forward.

### What are some alternatives to dbt-fal?
Unfortunately, none that we are aware of.

### I have more questions. Who can I talk to?

> Please feel free to contact us on our [discord](https://fal.ai/).

Author: Hesitant to add a direct discord link as it might get expired. also not sure we want to redirect people there generally. Maybe we should create an email smth like dbt@fal.ai ?

**Big thanks** to every single one of you who’s been with us on the dbt-fal adventure. Your support and contributions mean the world. We’ve done some awesome stuff together, and this isn’t the end. Just a new chapter. So, here’s to more amazing things ahead, and we’re stoked to have you join us for the ride.

Cheers!

## ❗ What Does This Mean?

- **No Further Development:** The project will no longer receive updates or new features.
- **Security Vulnerabilities:** We will not be addressing new security vulnerabilities after April 8, 2024. We advise users to consider this when deciding to continue the use of the project.
- **Archival:** The repository will be archived, making it read-only. While the code will remain accessible for educational and historical purposes, we encourage users to fork the repository if they wish to continue development on their own.

## 💬 FAQ

### Can I still use dbt-fal?
Yes, the project will remain available for use, but please be aware that no new updates or security patches will be provided moving forward.

### What are some alternatives to dbt-fal?
Unfortunately, none that we are aware of.

### I have more questions. Who can I talk to?

> Please feel free to contact us on our [discord](https://fal.ai/).

Author: ```suggestion
If you want to talk about dbt Python support, the best place to do so is the dbt Slack community. For other questions, reach out to hello@fal.ai
```

> If you want to talk about dbt Python support, the best place to do so is the dbt Slack community. For other questions, reach out to hello@fal.ai

Author: ```suggestion
If you want to talk about dbt Python support, the best place to do so is the [dbt Slack community](https://www.getdbt.com/community/join-the-community). For other questions, feel free to reach out to hello@fal.ai
```
